web/releases/nightly.json

```
@@ -39,15 +39,15 @@
           "path": "Managing_Hosts"
         },
         {
-          "title": "Configuring Hosts Using Ansible",
+          "title": "Configuring Hosts by Using Ansible",
```

Shouldn't this match exactly what's in `guides/common/attributes-titles.adoc`?
```suggestion
          "title": "Configuring hosts by using Ansible",
```

Or is that something we should fix separately since it looks like all others are using title case?